CONTRIBUTING.md

```diff
@@ -39,8 +39,10 @@ $ npm install
 
 ### Add Secret Keys
 
-1. In the cloned repo, create a file named `.env`. This will store environment variables you may want to keep secret.
-2. Create a variable named `DISCORD_SECRET_KEY` (you will assign this a value later on).
+1. The cloned repo includes a file named sample.env which contain the environmental variables. Copy this file to another file named .env by running the command below in the project main directory. Several of these variables need to be kept secret so this file is not commited to github.
```

Comment:
A few minor nit-picks:

- "sample.env which contain" ➡️  "sample.env which contains"
- "so this file is not commited to github" ➡️ "so the `.env` file is not committed to GitHub"

Reply:
I made those changes and added a script to package.json which deploys the application commands.